azurerm/helpers/azure/app_service.go

```diff
@@ -316,6 +316,17 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 								Optional:     true,
 								ValidateFunc: validation.StringIsNotEmpty,
 							},
+							"name": {
+								Type:         schema.TypeString,
+								Optional:     true,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+							"priority": {
+								Type:         schema.TypeInt,
+								Optional:     true,
+								Computed:     true,
```

Comment:
as @sebader noted, priority defaults to 65000 when not provided. Should we make that an explicit default or let it be computed?

Reply:
I'm not sure where the default of 65000 is injected in this case. Might be in the Go SDK? Since in CLI, Powershell and the portal the parameter must be manually specified, there is no default there.

Reply:
So might actually be better so explicitly set it?!